scripts/release.sh

```diff
@@ -3,6 +3,7 @@ set -euo pipefail
 
 python manage.py migrate --noinput
 python manage.py import_shapes data/final/shapes/chicago_shapes.json
+python manage.py populate_person_statistics
```

Comment:
we are also going to need to run this as part of https://github.com/datamade/chicago-council-scrapers/

Reply:
cool - i'll make a PR there